pylint/checkers/base.py

```diff
@@ -2270,6 +2270,32 @@ def _check_docstring(
             )
 
 
+class EllipsisChecker(_BasicChecker):
```

Comment:
Please create a new file ``pylint/checker/ellipsis_checker.py``, as we want to burst base.py in #5489